packs/tfe_fdo_nomad/templates/tfe.agent.nomad.tpl

```diff
@@ -0,0 +1,57 @@
+# Copyright (c) HashiCorp, Inc.
+# SPDX-License-Identifier: MPL-2.0
+
+job "tfe-agent-job" {
```

Comment:
Why is the main job name configurable but this one isn't?

Reply:
The design is such that the only job name we permit for TFE agents is `tfe-agent-job`.

Reply:
Weird, but ok.